torch/_inductor/codegen/common.py

```diff
@@ -2413,7 +2443,7 @@ def maybe_append_choice(self, choices, **kwargs):
         except NotImplementedError as e:
             return e
```

Comment:
Ugh, we are really suppose to return the Error? NotImplementedError and NotImplemented are two different classes. Seems like this could be a bug

Reply:
I agree, but this is not something changed in this PR.